### PR TITLE
Add Italian language

### PR DESCRIPTION
**Pull request by Luni-4**
- Fix some English words in order to simplify the translation process to the Italian language and sort them out
- Implement code to add more languages to `scribble.rs`, in my case the Italian language
- Create a `Python` script that automatically adds tags at the end of the words

Thanks in advance for your review! :)

### CODE REVIEW

**Comment by hija** [comment]
I think it's a great addition that you can chose the language in the lobby setup. :+1: 

Regarding the Python script: I am not sure if adding a python script into a go repository is reasonable. It might confuse people and the same program could possibly be achieved in go.

#JustMyTwoCents

File: lobby.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ var (
 		MinClientsPerIPLimit: 1,
 		MaxClientsPerIPLimit: 24,
 	}
+	supportedLanguages = []string{"English", "Italian"}
```
Comment (Bios-Marcel) [inline lobby.go]: These should be constants, as they are also used at:

https://github.com/scribble-rs/scribble.rs/blob/8d4c820afe821dff9f37579644346191036a9376/words.go#L16

File: lobby.go [hunk 2]
```diff
@@ -845,6 +850,9 @@ func CreateLobby(w http.ResponseWriter, r *http.Request) {
 	clientsPerIPLimit, clientsPerIPLimitInvalid := parseClientsPerIPLimit(r.Form.Get("clients_per_ip_limit"))
 	enableVotekick := r.Form.Get("enable_votekick") == "true"
 
+	// Read wordlist according to the chosen language
+	readWordList(language)
```
Comment (Bios-Marcel) [inline lobby.go]: Every lobby will cause a read of the wordlist. All wordlists should be read once on startup and then never again.

File: resources/words/add_tag.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```
Comment (Bios-Marcel) [inline resources/words/add_tag.py]: Could you please explain what exactly this script does and how to use it? Which kind of input does it expect and what's the actual gain? I thought it auto-translates, but I see no translation code :man_shrugging:

**Comment by Bios-Marcel** [comment]
I also agree with what @hija said, I am kind of against mixing the tooling unnecessarily.

File: resources/words/add_tag.py (hunk 3 above)
Comment (hija) [reply resources/words/add_tag.py]: I think the italian word file is a 1-to-1 translation of the english file. So the first line of the italian translation matches the first word of the english file etc. This script adds - as far as I understand it - the difficulty level (#h etc.) at the end.

Comment (Bios-Marcel) [reply resources/words/add_tag.py]: But there's no actual translation happening in this script, right? Or am i blind?

Comment (hija) [reply resources/words/add_tag.py]: As far as I understand it - no there is not. It takes an already translated language file and just adds the tag at the end.

**Comment by Luni-4** [comment]
@hija @Bios-Marcel 

I agree with you, I will upload the same version of the script in Python. That script doesn't automatically translate words, it only adds tags at the end of the word, perhaps I could expand that.

Thanks for your advice! :)

File: resources/words/add_tag.py (hunk 3 above)
Comment (Luni-4) [reply resources/words/add_tag.py]: I will make another PR for this script eventually

File: lobby.go (hunk 2 above)
Comment (Luni-4) [reply lobby.go]: How can I do that without using the `init` function? Because I need to pass the language chosen by the user.

Comment (Bios-Marcel) [reply lobby.go]: actually, nevermind, this is better

**Comment by Bios-Marcel** [comment]
Could you stop force-pushing please? This way i can't see the changes between your pushes.

**Comment by Luni-4** [comment]
> Could you stop force-pushing please? This way i can't see the changes between your pushes.

The first time I force pushed because I had to remove a file and the second time because there was just a single change, don't worry, if more changes are necessary I wouldn't force push :)

**Pull request merged**